stable/artifactory-ha/values.yaml

```diff
@@ -1245,8 +1247,12 @@ nginx:
         proxy_pass_header   Server;
         proxy_cookie_path   ~*^/.* /;
         proxy_pass          http://{{ include "artifactory-ha.fullname" . }}:{{ .Values.artifactory.externalPort }}/;
+        {{- if .Values.nginx.service.ssloffload}}
+        proxy_set_header    X-Artifactory-Override-Base-Url $http_x_forwarded_proto://$host/artifactory;
```

Comment:
Need to change to the new header: `X-JFrog-Override-Base-Url` and also remove the `/artifactory`.

Reply:
Changed the header and removed the context